content/en/docs/languages/python/libraries.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Using instrumentation libraries
+linkTitle: Libraries
+weight: 40
+cSpell:ignore: HTTPX httpx instrumentor uninstrument
+---
+
+{{% docs/languages/libraries-intro "Python" %}}
+
+## Use instrumentation libraries
+
+If a library isn't included with OpenTelemetry, you can use
+[instrumentation libraries](/docs/specs/otel/glossary/#instrumentation-library)
+to generate telemetry data for a library or framework.
+
+For example,
+[the instrumentation library for HTTPX](<[https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNetCore](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation/opentelemetry-instrumentation-httpx)>)
+automatically creates [spans](/docs/concepts/signals/traces/#spans) based on
+HTTP requests.
+
+## Setup
+
+You can install each instrumentation library separately using pip. For example:
+
+```sh
+pip install opentelemetry-instrumentation-{integration}
+```
+
+In the previous example, `{integration}` is the name of the instrumentation. For
+a complete list of instrumentations and their names, browse the
+[instrumentation](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation)
+directory of the `opentelemetry-python-contrib` repository on GitHub.
+
+To install a development version, clone or fork the
+`opentelemetry-python-contrib` repository and run the following command to do an
+editable installation:
+
+```sh
+pip install -e ./instrumentation/opentelemetry-instrumentation-{integration}
+```
+
+## Example with HTTPX instrumentation
+
+Here's how you can instrument HTTP requests made using the `httpx` library.
+
+First, install the instrumentation library using pip:
+
+```sh
+pip install opentelemetry-instrumentation-httpx
+```
+
+Next, use the instrumentor to automatically trace requests from all clients:
+
+```python
+import httpx
+from opentelemetry.instrumentation.httpx import HTTPXClientInstrumentor
+
+url = "https://some.url/get"
+HTTPXClientInstrumentor().instrument()
+
+with httpx.Client() as client:
+     response = client.get(url)
+
+async with httpx.AsyncClient() as client:
+     response = await client.get(url)
+```
+
+If needed, you can uninstrument specific clients or all clients. For examples:
```

Comment:
I'm a little wary about this here because folks might not read the "If needed" part and just copy-paste `uninstrument` calls.

Maybe it's worth putting in an explicit section marked as optional? Because in practice, you don't need to actually uninstrument unless you're doing something very specific (like testing your own instrumentation library).

Reply:
Chose to create a subsection for this.